radixconfig.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: radix.equinor.com/v1
+kind: RadixApplication
+metadata:
+  name: flotilla
+spec:
+  build:
+    secrets:
+      - TLS_SERVER_KEY
+  environments:
+    - name: dev
+      build:
+        from: main
+    - name: qa
+    - name: prod
+  components:
+    - name: frontend
+      src: frontend
+      dockerfileName: Dockerfile.Radix
```

Comment:
Do we need a seperate dockerfile for the radix or can we use/adapt the existing dockerfile for the frontend?

Reply:
Or can I assume that is what is explained at line 8 in the file?

Reply:
You may assume that yes